Sources/NiceComponents/Components/NiceImageResource.swift

```diff
@@ -0,0 +1,10 @@
+//
```

Comment:
Is this still work in progress?

Reply:
Oops, leftover from a previous try. removed now.